python/cac_tripplanner/templates/cac_tripplanner/cac_tripplanner_home.html

```diff
@@ -0,0 +1,79 @@
+{% extends "cac_tripplanner/base_cac_tripplanner.html" %}
```

Comment:
For the size of this app, and to follow typical django convention, these templates should probably just be:
```
python/cac_tripplanner/templates/base.html
python/cac_tripplanner/templates/home.html
```
If you need a subfolder to namespace your templates, you should just use different django apps instead.